drivers/net/ethernet/renesas/rswitch.c

```diff
@@ -3106,6 +3109,9 @@ static int rswitch_gwca_hw_init(struct rswitch_private *priv)
 	priv->gwca.speed = 1000;
 	rswitch_gwca_set_rate_limit(priv, priv->gwca.speed);
 
+	/* Set IRQ delay base period to max value */
+	rs_write32(0x7ff, priv->addr + GWIDPC);
```

Comment:
NIT: you can add define like
`#define IRQ_MAX_PRESCALER_VAL (0x7fff)`
to avoid using magic numbers.

Reply:
Thanks for RB, fixed. The same was made for GWCA_IRQ_DELAY_MASK